<launch>
  
  <node pkg="naos_localization" type="main" name="main" output="screen">
    <param name="camera" value="0"/>

    <param name="H_MIN" value="115"/>
    <param name="H_MAX" value="180"/>
    <param name="S_MIN" value="100"/>
    <param name="S_MAX" value="255"/>
    <param name="V_MIN" value="90"/>
    <param name="V_MAX" value="255"/>
  </node>

</launch>
